workflows/genetic-demux/map-sc-star.nf

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+STARCONTAINER = 'quay.io/biocontainers/star:2.7.9a--h9ee0642_0'
+SAMTOOLSCONTAINER = 'quay.io/biocontainers/samtools:1.14--hb421002_0'
```

Comment:
It might be worth pulling these out into the config file since you are using them here and in the bulk workflow.

Reply:
Yes, eventually, but the mode for this repo has been so far for things to be mostly independent, so I wasn't doing that quite yet... Expect it in a later revision when I try to tie all the parts together!